src/test/scala/chiselTests/Direction.scala

```diff
@@ -379,4 +381,50 @@ class DirectionSpec extends ChiselPropSpec with Matchers with Utils {
     assert(emitted.contains("io.monitor.valid <= io.driver.valid"))
     assert(emitted.contains("io.monitor.ready <= io.driver.ready"))
   }
+  property("Bugfix: marking Vec fields with mixed directionality as Output clears inner directions") {
+    class Decoupled extends Bundle {
+      val bits = UInt(3.W)
+      val valid = Bool()
+      val ready = Flipped(Bool())
+    }
+    class DecoupledAndMonitor extends Bundle {
```

Comment:
i dont get the name of this class. For clarity should this be `DecoupledMonitor` and the directon be an Input?

Reply:
I was just using old code from a different PR, I'll rename it.